root/static/scripts/common/utility/arrays.js

```diff
@@ -122,6 +122,10 @@ export function groupBy<T>(
   }, Object.create(null));
 }
 
+export function head<T>(array: ?$ReadOnlyArray<T>): ?T {
+  return array?.length ? array[0] : undefined;
+}
+
```

Comment:
Maybe I'm just being boring, but if the last is "last", why can't we name the first "first"? :p